app/src/main/java/org/thoughtcrime/securesms/components/ProfilePictureView.kt

```diff
@@ -111,7 +111,7 @@ class ProfilePictureView : RelativeLayout {
             val avatar = (signalProfilePicture as? ProfileContactPhoto)?.avatarObject
             if (signalProfilePicture != null && avatar != "0" && avatar != "") {
                 glide.clear(imageView)
-                glide.load(signalProfilePicture).diskCacheStrategy(DiskCacheStrategy.AUTOMATIC).circleCrop().into(imageView)
+                glide.load(signalProfilePicture).diskCacheStrategy(DiskCacheStrategy.NONE).circleCrop().into(imageView)
```

Comment:
How does this affect performance?

Reply:
can probably leave this out, just cleaning up references to these as I think they contribute to caching decoded images in a temp app directory instead of generating them or loading from the DB or memory, but can take a look at that in a separate PR